src/main/java/org/launchcode/tutorconnector/models/AbstractEntity.java

```diff
@@ -0,0 +1,90 @@
+package org.launchcode.tutorconnector.models;
+
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.MappedSuperclass;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+import java.util.Objects;
+
+@MappedSuperclass
+public class AbstractEntity {
```

Comment:
abstract class no constructors

Reply:
I believe that abstract classes are able to have constructors.